### PR TITLE
docs: user tutorial for GCP NFS

### PR DESCRIPTION
**Pull request by abalaie**
**Description**

Changes proposed in this pull request:

- user tutorial for GCP NFS
- small changes to glossary
- small changes to gcpNfsVolume CRD document

**Related issue(s)**

### CODE REVIEW

**Comment by abalaie** [comment]
Updated the document to be in sync with recent changes.

File: docs/user/tutorials/01-20-gcp-nfs-volume.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+   This workload should print a sequence of 20 lines to stdout and a file on the NFS volume.
+   Then it should print the content of the file.
+   
+7. Print the logs of one of the workloads, run:
+
+   ```shell
+   kubectl logs -n $NAMESPACE `kubectl get pod -n $NAMESPACE -l app=gcpnfsvolume-demo -o=jsonpath='{.items[0].metadata.name}'`
+   ```
+   
+   The command should print something like:
```
Comment (mmitoraj) [inline docs/user/tutorials/01-20-gcp-nfs-volume.md:142]:
```suggestion
   The command should print a result similar to the following one:
```

File: docs/user/tutorials/01-20-gcp-nfs-volume.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    Hello from gcpnfsvolume-demo-557dc8cbcb-vg5zt: 1
+    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 1
+    Hello from gcpnfsvolume-demo-557dc8cbcb-vg5zt: 2
+    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 2
+    Hello from gcpnfsvolume-demo-557dc8cbcb-vg5zt: 3
+    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 3
+   ...
+   ```
+   
+   Note that the content after `File content:` contains prints from both workloads. This
```
Comment (mmitoraj) [inline docs/user/tutorials/01-20-gcp-nfs-volume.md:157]:
```suggestion
   >[!NOTE] The `File content:` contains prints from both workloads. This demonstrates the ReadWriteMany capability of the volume.
```

File: docs/user/tutorials/01-20-gcp-nfs-volume.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 1
+    Hello from gcpnfsvolume-demo-557dc8cbcb-vg5zt: 2
+    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 2
+    Hello from gcpnfsvolume-demo-557dc8cbcb-vg5zt: 3
+    Hello from gcpnfsvolume-demo-557dc8cbcb-kwwjt: 3
+   ...
+   ```
+   
+   Note that the content after `File content:` contains prints from both workloads. This 
+   demonstrates the ReadWriteMany capability of the volume.
```
Comment (mmitoraj) [inline docs/user/tutorials/01-20-gcp-nfs-volume.md:158]:
```suggestion
```

File: docs/user/tutorials/01-20-gcp-nfs-volume.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+   You should see the following message:
+
+   ```
+   NAME     STATUS   VOLUME     CAPACITY   ACCESS MODES   STORAGECLASS 
+   my-vol   Bound    {PV_NAME}  1Ti        RWX                                                
+   ```
+
+   Similarly to PV, note the `RWX` access mode and `Bound` status.
+
+6. Create two workloads that both write to the volume
```
Comment (mmitoraj) [inline docs/user/tutorials/01-20-gcp-nfs-volume.md:72]:
```suggestion
6. Create two workloads that both write to the volume. Run:
```

**Review by mmitoraj — CHANGES_REQUESTED** [review]
Please also add the tutorial to the list in `_sidebar.md`.

**Comment by abalaie** [comment]
I rebased the code as it had some conflicts with the base and the option was not available on the UI. 
But then it showed many commits that were already merged to the "main" branch. So based on Github suggestions on the same issue, I changed the base branch to something else and again back to main, so that the base branch commit get updated and it only shows the relevant changes. And it worked.

**Review by tmilos77 — APPROVED** [review]

**Pull request merged**